modules/mockserver/src/test/java/org/testcontainers/containers/MockServerContainerRuleTest.java

```diff
@@ -12,7 +12,7 @@
 public class MockServerContainerRuleTest {
 
     public static final DockerImageName MOCKSERVER_IMAGE = DockerImageName.parse(
-        "jamesdbloom/mockserver:mockserver-5.5.4"
+        "jamesdbloom/mockserver:mockserver-5.13.2"
```

Comment:
Just curious why we had to update this one?

Reply:
mockserver dependency expects two additional fields that are not provided by an old image. I remember that it is recommended to use the same version for the jar and docker image